Manifests/ManagedPreferencesApplications/com.secondsonconsulting.baseline.plist

```diff
@@ -431,6 +431,82 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_type</key>
 			<string>array</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>2.2</string>
+			<key>pfm_description</key>
+			<string>Define files which you want Baseline to wait for. Use this for items not directly installed by Baseline, like VPP or MDM installed apps.</string>
+			<key>pfm_name</key>
+			<string>WaitFor</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_description</key>
+							<string>The name you want to appear in the SwiftDialog menu for this item we are waiting for.</string>
+							<key>pfm_name</key>
+							<string>DisplayName</string>
+							<key>pfm_title</key>
+							<string>Display Name</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_description</key>
+							<string>A path to the file you want Baseline to wait for.</string>
+							<key>pfm_name</key>
+							<string>Path</string>
+							<key>pfm_title</key>
+							<string>Path</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_description</key>
+							<string>The icon you want to appear in the SwiftDialog menu for this item. Can be a URL or local file path.</string>
+							<key>pfm_name</key>
+							<string>Icon</string>
+							<key>pfm_title</key>
+							<string>Icon</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_app_min</key>
+							<string>2.1</string>
+							<key>pfm_description</key>
+							<string>The subtitle text to appear on this line item.</string>
+							<key>pfm_name</key>
+							<string>Subtitle</string>
+							<key>pfm_title</key>
+							<string>Subtitle</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+					</array>
+					<key>pfm_type</key>
+					<string>dictionary</string>
+				</dict>
+			</array>
+			<key>pfm_title</key>
+			<string>WaitFor</string>
+			<key>pfm_type</key>
+			<string>array</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>2.2</string>
+			<key>pfm_description</key>
+			<string>When using WaitFor, enter the number of seconds you wish to wait before Baseline considers remaining items a failure. This timer does not start until all other items are processed. Default 300.</string>
```

Comment:
If you replace the default value mention in the description with a `pfm_default` element you'll get a better presentation.

Reply:
Done!